src/styles/main.scss

```diff
@@ -1,3 +1,74 @@
+$cell-size: 100px;
+
+html {
+  font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.field {
+  display: flex;
+  justify-content: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $cell-size * 7 + 6px;
+  border: 1px solid #000;
+  padding: 10px;
+  gap: 1px;
+
+  & :first-child {
+    margin-left: $cell-size * 6 + 6px;
```

Comment:
```suggestion
  & :first-child {
    margin-left: $cell-size * 6 + 6px;
```
margin-left should be flexible and depends on start-day modifier
no modifier does not work correctly
![Screenshot from 2023-05-21 16-08-03](https://github.com/mate-academy/layout_calendar/assets/33029864/44447cbb-ce1a-4a0d-aced-516a54b8e491)
![Screenshot from 2023-05-21 16-07-40](https://github.com/mate-academy/layout_calendar/assets/33029864/b6d08e2a-6663-431a-bf54-b096837de7fc)